src/content/self-hosted/install/upgrade.mdx

```diff
@@ -65,6 +65,43 @@ ingress-nginx:
 
 > Keeping the same `issuer` and `certificate` reduces moving parts during the upgrade, but we recommend you to [bring your own certificate](self-hosted/administration/certificates.mdx) once the Okteto upgrade has been complete succesfully.
 
+As part of [1.7.0 version](self-hosted/install/releases.mdx#170), we moved the configuration settings needed to configure the external storage for the registry. Before, it was configured under `cloud` key and now it has to be configured under `registry` key.
```

Comment:
```suggestion

## Upgrading to 1.7.x

As part of [1.7.0 version](self-hosted/install/releases.mdx#170), we moved the configuration settings needed to configure the external storage for the registry. Before, it was configured under `cloud` key and now it has to be configured under `registry` key.
```

Reply:
@rberrelleza If we set the title there, it would look like a bit odd the rest of the subsections for `upgrading tp 1.x`. I mean, we currently have :

## Upgrading to 1.x

loren ipsum

#### Before upgrade

loren ipsum

#### Upgrade Okteto

loren ipsum

#### After upgrade

loren ipsum

-----

If we add that title there, it would look like this:

## Upgrading to 1.x

loren ipsum

#### Before upgrade

loren ipsum

## Upgrading to 1.7.x

loren ipsum

#### Upgrade Okteto

loren ipsum

#### After upgrade

loren ipsum

---- 

`Upgrade Okteto` and `After upgrade` sections refers to the changes mentioned in the `upgrading to 1.x`

That's why I had doubts about the best way of including that information. Adding a subsection for `1.7` might make the upgrade section looks a bit misleading

Reply:
ok, thanks!